README.md

```diff
@@ -154,6 +154,30 @@ for peer <- [subscriber, publisher], do: Spell.close(peer)
 
 See `Spell.Role.Publisher` and `Spell.Role.Subscriber` for more information.
 
+#### Pattern-based subscriptions
+
+By default, peers subscribe to topics using an exact matching policy. Using
+pattern-based subscriptions a peer can receive messages that match certain
+criteria.
+
+```elixir
+# assuming we have a subscriber peer connected
+
+# exact match
+{:ok, subscription} = Spell.call_subscribe(subscriber, "com.spell.my_topic")
+# subscriber will receive messages published in the "com.spell.my_topic" topic
+
+# prefix match
+{:ok, subscription} = Spell.call_subscribe(subscriber, "com.spell.my_topic_prefix", options: %{match: :prefix})
```

Comment:
Ah, yeah -- the justification for having an specific `options` keyword arg is that it maps directly to the WAMP message's `options` argument. I agree, this is a good discussion to open up.